packages/flame/lib/src/cache/assets_cache.dart

```diff
@@ -73,6 +73,25 @@ class AssetsCache {
     final bytes = Uint8List.view(data.buffer);
     return _BinaryAsset(bytes);
   }
+
+  /// This method provides synchronous access to cached assets, similar to
+  /// [Images.fromCache].
```

Comment:
This is referring to the wrong file.
```suggestion
  /// [AssetsCache.fromCache].
```